src/receive_imf.rs

```diff
@@ -1721,11 +1727,6 @@ async fn is_probably_private_reply(
         }
     }
 
-    let is_reaction = mime_parser.parts.iter().any(|part| part.is_reaction);
-    if is_reaction {
-        return Ok(false);
-    }
-
     Ok(true)
```

Comment:
I checked, for usual outgoing chat messages we get to here, it seems that for reactions it should be the same then